### PR TITLE
Sickness event fixes

### PR DESCRIPTION
**Pull request by ThatDan123**
CL:Fixes Random Sickness Events

Also fixes issue with admin spawned sickness parsing ints

### CODE REVIEW

File: UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSickness.cs [hunk 1]
```diff
@@ -31,14 +32,32 @@ public override void OnEventEndTimed()
 
 		private static void SpawnSickness(string serializedEventParameters)
 		{
-			SicknessEventParameters sicknessEventParameters = JsonConvert.DeserializeObject<SicknessEventParameters>(serializedEventParameters);
+			SicknessEventParameters sicknessEventParameters;
+
+			//If null means its a server random event so has no default parameters
+			//Todo: when infection system is more deadly change random effects based on player numbers
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSickness.cs]: very stupid but can you change these to ``TODO:`` in all caps. Some IDE's detect them only when they are formatted like that.

**Comment by corp-0** [comment]
also I don't know if it has been fixed already but can you make allergies less frequent?

edit: If that is not part of the event system then it is out of the scope and you can ignore this comment.

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSickness.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6325130&bid=20556045)

**Comment by ThatDan123** [comment]
Dont think allergies are a part of the random event.

**Comment by corp-0** [comment]
Good, then it is good to merge after tests.

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/Systems/InGameEvents/InGameEventScripts/EventSickness.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6325130&bid=20556045)

**Pull request merged**